hadoop-hdds/interface-client/pom.xml

```diff
@@ -143,38 +132,21 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         <version>${maven-antrun-plugin.version}</version>
         <executions>
           <execution>
+            <goals>
+              <goal>run</goal>
+            </goals>
             <phase>generate-sources</phase>
             <configuration>
               <target>
-                <replace token="com.google.protobuf"
-                         value="org.apache.ratis.thirdparty.com.google.protobuf"
-                         dir="target/generated-sources/java/org/apache/hadoop/hdds/protocol/datanode/proto">
```

Comment:
I find the original format more readable, but the option in `sortpom` to preserve this (`indentAttribute`) is available only in newer versions that do not support Java 8.

Reply:
Me too. The original format was easy to read.
Let's revisit this once we move to the newer version.